src/app/modules/interceptors/auth.interceptor.ts

```diff
@@ -39,15 +40,21 @@ export class AuthInterceptor implements HttpInterceptor {
         headers: request.headers.append('Authorization', `Bearer ${this.localStorageManager.getAuthenticationInfo()?.token}`)
       });
     }
+
     return next.handle(request).pipe(
-      catchError((error: HttpErrorResponse) => {
-        const unauthorized = error.status === 401;
+      map((response: any) => {
+        const noError = response.status === 200;
```

Comment:
write logic with switch

Reply:
avoid any! you try getting .status frim any type, check that it really HttpResponse